robottelo/vm.py

```diff
@@ -302,6 +304,8 @@ def register_contenthost(self, org, activation_key=None, lce=None,
                 username,
                 password,
             )
+            if auto_attach:
+                cmd += u' --auto-attach'
```

Comment:
No it does not work with with org+activation key, subscription-manager will raise an exception
when using org+activation key you can use command:
```console
subscription-manager attach --auto
```